src/modules/AltAttribute.js

```diff
@@ -0,0 +1,48 @@
+import { BaseModule } from './BaseModule'
+
+export class AltAttribute extends BaseModule {
+  onCreate = () => {
+    // Create the container to hold the input element
+    this.display = document.createElement('div')
+
+    // Apply styles
+    Object.assign(this.display.style, this.options.displayStyles)
+
+    // Attach it
+    this.overlay.appendChild(this.display)
+
+    // ALT input
+    const input = document.createElement('input')
+    if (this.img.alt) {
+      input.value = this.img.alt
+    }
+    input.addEventListener('input', (e) => {
+      this.img.alt = e.target.value
+    })
+
+    this.display.insertAdjacentHTML('beforeend', 'Alt: ')
+    this.display.appendChild(input)
+  }
+
+  onDestroy = () => {}
```

Comment:
ARE you sure we don't need to clean up those event listeners here @Misticek ?
Seems like that could be a little memory leak, unless the input is removed from the DOM on destroy obviously.

Reply:
![](https://media0.giphy.com/media/VgT9bySXhoucgkgNOf/giphy.gif)

Reply:
you might want to store your DOM node on `this`
ideally as `this.altInput` or something like that

Reply:
yep I know, I accidentally named it same as you suggested 😄